docs/src/notation.md

```diff
@@ -0,0 +1,24 @@
+# Notation overview
+
+Since manifolds include a reasonable amount of elements and functions, the
+following list tries to keep an overview of used notation throughout `Manifolds.jl`.
+The order by terminology. Thy might be used in a plain form within the code or
+when referring to that code. This is for example the case the caligraphic symbols.
+
+Within the documented functions the utf8 symbols are used whenever possible,
+as long as that renders still in TeX within this documentation.
+
+A
+
+| Symbol | Description | Also used | Comment |
+|:--:|:-------------- |:--:|:--- |
+| ``T^*_x ℳ`` | The cotangent space at ``x`` | | |
+| ``ξ`` | A cotangent vector from ``T^*_x ℳ`` | ``ξ_1,ξ_2,…,η,𝛇`` | |
+| ``F`` | A fiber | |
+| ``⟨·,·⟩`` | inner product (in ``T^*_x ℳ``) | ``⟨·,·⟩_x, g_x(·,·)`` |
```

Comment:
Can you verify that `⟨` looks the same as `\langle` when rendered, including spacing? Looks a bit cramped in the docs.

Reply:
You mean `T_x ℳ` right? The tangent space?.
The notation on the left and right are different.
There's also Einstein notation which is convenient for some of the metric functions, but I don't think we need to document that here.

Reply:
Well it looked cramped because it was not TeXed, see comment below for markdown files and within tables let's got back to TeX instead of utf8. Oh and yes the inner product is on ´T_xM` of course.